Workiva/w-mobile-kit
# MOBILE-1625: Create pod and project for w-mobile-kit repo

## Description

Setup the project and pod parts for the new w-mobile-kit repo.
## What Was Changed

Added the pod configuration files, the sample project, and a sample file for the library.
## How to Test
1. Follow the README to initialize and install the library into an existing project.
2. Ensure the pod file is successfully pulled in and is shown with the sample file in the project. 
## 

@Workiva/mobile
